.gitignore

```diff
@@ -8,3 +8,5 @@ venv/
 **__pycache__/
 *.pyc
 **db.sqlite3
+media/
```

Comment:
The `media/` directory is included in the `.gitignore` file. If you intend to version control media files, you should remove this line. Otherwise, ensure that excluding this directory is intentional.